apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/transaction/messages/AppendixApplierRegistry.java

```diff
@@ -14,14 +14,20 @@
 public class AppendixApplierRegistry {
     private final Map<Class<?>, AppendixApplier<?>> appliers = new HashMap<>();
 
-    public AppendixApplierRegistry() {}
+    public AppendixApplierRegistry() {
+    }
 
 
     void init(Instance<AppendixApplier<?>> instances) {
-        instances.iterator().forEachRemaining(e-> appliers.put(e.forClass(), e));
+        instances.iterator().forEachRemaining(e -> appliers.put(e.forClass(), e));
+    }
+
+    void addApplier(AppendixApplier<?> applier) {
```

Comment:
AppendixApplier's must be registered automatically using CDI instances

Reply:
removed